index.md

```diff
@@ -49,9 +49,18 @@ $ $KAFKA_HOME/bin/kafka-console-producer.sh --topic=topic \
 
 Start another shell and start a consumer:
 
-```
-$ $KAFKA_HOME/bin/kafka-console-consumer.sh --topic=topic --zookeeper=$ZK
-```
+- With later version of kafka (0.10.x ~ latest)
+
+	```
+	$ $KAFKA_HOME/bin/kafka-console-consumer.sh --topic=topic --bootstrap-server=`broker-list.sh`

```

Comment:
This should be 0.9 to latest with `--new-consumer`

Reply:
look these two link with tag:
[kafka 0.10.0 tag quickstart_consume](http://kafka.apache.org/0100/documentation.html#quickstart_consume)
[kafka 0.10.1 tag quickstart_consume](http://kafka.apache.org/0101/documentation.html#quickstart_consume)

i think new consumer api had been used as default api in 0.10.1 without `--new-consumer`
. As the links wroten, `--zookeeper` option is default for versions <= 0.10.0, and `--bootstrap-server` is default for versions >= 0.10.1.

Reply:
I would consult the source code for ConsoleConsumer rather than the html

Reply:
i find the latest tag of version 0.10.0 (0.10.0.1-rc2) and ealiest tag of version 0.10.1 (0.10.1.0) , 


ver 0.10.0.1-rc2

https://github.com/apache/kafka/blob/a7a17cdec9eaa6c57cc634da6a7c766d98d4160e/core/src/main/scala/kafka/tools/ConsoleConsumer.scala#L247

ver 0.10.1.0

https://github.com/apache/kafka/blob/3402a74efb23d1d4fef1e7284de8d4b151cde784/core/src/main/scala/kafka/tools/ConsoleConsumer.scala#L290

i think new consumer is the default option for the versions equal to and after 0.10.1.